frontend/src/framework/theme/styles/colors.styles.ts

```diff
@@ -7,9 +7,8 @@ const colors = {
         50: '#e2e1ec',
     },
     brand: {
-        900: '#1a365d',
-        200: '#b3e0ff',
```

Comment:
Don't delete these colors because they are used in the Button component.

Reply:
But we don't need them in Button because we will have other colors there. In the figma design we don't have these colors at all.

Reply:
In that case, okay. You can delete them.